solutions/silver/usaco-1254.mdx

```diff
@@ -37,7 +37,7 @@ vector<pll> orders_to[MAX_N];  // Stores for each barn its outgoing orders
 
 int bales[MAX_N];
 bool visited[MAX_N];
-int n, cnt;
+int n, cnt;  // cnt is the number of total orders we need to make (in an optimal solution)
```

Comment:
ok in this case just name cnt to `order_num`
also declare it separately from `n`
sorry for the last-minute stuff, i swear this is iit

Reply:
np, should be good now